### PR TITLE
CF_W4_Dict&Set_lab

### PR DESCRIPTION
**Pull request by Misseswaddles**

### CODE REVIEW

**Comment by Misseswaddles** [comment]
Adding W2 series.py completed fibonacci exercise to pull request

File: students/CarolF/module02/series.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+                count = 1
+                currentNumb = 1
+                previousNumb = 0
+            if i >1:
+                count =  previousNumb + currentNumb
+                previousNumb= currentNumb
+                currentNumb = count
+
+    return(count)
+
```
Comment (summerela) [inline students/CarolF/module02/series.py]: You can also do something like: 

    return fibonacci(n - 1) + fibonacci(n - 2)

**Comment by summerela** [comment]
Looks good so far! Have you made any progress with the mailroom assignments?

**Comment by Misseswaddles** [comment]
Hi Summer,

I have been working on them. I am wrapping up the slicing lab--I just
checked my answers with Maria's test.py and realized I way over-coded my
solution. So, I am going to edit my solution and make sure i know it works.
Then I will do the lab.

It was very unwise of me to try to take two certification programs at once!
However, I am determined to catch up and get going! I have reserved extra
time this week so I am hoping I will be able to catch up by
Wednesday...fingers crossed.

Best,
Carol Farris

**Pull request merged**